### PR TITLE
fix(doc): check for "0" in if condition

### PR DESCRIPTION
**Pull request by P-Miranda**
- `if [ $(RESULTS) ]` is evaluated as true even if `$(RESULTS)=0`

### CODE REVIEW

File: document/Makefile [hunk 1]
```diff
@@ -63,10 +63,10 @@ ugtop.tex: texfiles
 
 doctop:
 	echo "\def\NAME{$(NAME)}" >> $(DOC)top.tex
-	if [ $(RESULTS) ]; then echo "\def\RESULTS{Y}" >> $(DOC)top.tex; fi
-	if [ $(INTEL_FPGA)$(AMD_FPGA) ]; then echo "\def\FPGA{Y}" >> $(DOC)top.tex; fi
-	if [ $(AMD_FPGA) ]; then echo "\def\AMD{Y}" >> $(DOC)top.tex; fi
-	if [ $(INTEL_FPGA) ]; then echo "\def\INTEL{Y}" >> $(DOC)top.tex; fi
+	if [ $(RESULTS) -ne 0]; then echo "\def\RESULTS{Y}" >> $(DOC)top.tex; fi
```
Comment (jjts) [inline document/Makefile:66]: why do I have to define RESULTS and the others ?

Comment (P-Miranda) [reply document/Makefile:66]: example cache `documentation.mk` sets variable to 0:
https://github.com/IObundle/iob-cache/blob/build-lib/document/document.mk

Comment (P-Miranda) [reply document/Makefile:66]: the alternative is to remove setting these variables to 0 in `document/document.mk` file

Comment (jjts) [reply document/Makefile:66]: that seems a lot better 

we do not add crap to cover up other crap: we fix the crap that is causing the problem

**Pull request closed without merging**